components/ide-metrics-api/idemetrics.proto

```diff
@@ -56,6 +56,8 @@ message ReportErrorRequest {
     string workspace_id = 5;
     string instance_id = 6;
     map<string, string> properties = 7;
+    string name = 8;
```

Comment:
I though we will just use properties? In this case should we change it to `error_name` or `error_message? the trouble here that we also getting very specific to JS in Java and golang it would be different?